docs/source/app-dev/parties-users.rst

```diff
@@ -0,0 +1,88 @@
+.. Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+Parties and Users On a Daml Ledger
+##################################
+
+Identifying parties and users is an important part of building a workable Daml application. Recall these definitions from the :doc:`Getting Started Guide <getting-started/app-architecture.html>`:
+
+- **Parties** are unique across the entire Daml network. These must be allocated before you can use them to log in, and allocation results in a random-looking (but not actually random) string that identifies the party and is used in your Daml code. Parties are a builtin concept.
+
+- On each participant node you can create **users** with human-readable user ids. Each user can be associated with one or more parties allocated on that participant node, and refers to that party only on that node. Users are a purely local concept, meaning you can never address a user on another node by user id, and you never work with users in your Daml code; party ids are always used for these purposes. Users are also a builtin concept.
+
+This represents a change from earlier versions of Daml, and the differences between versions are discussed in more depth here.
+
+Parties in SDK 1.18 and Previous
```

Comment:
do we really want to do a 1:1 copy here? Talking about the switch from 1.18 to 2.0 makes sense for a blogpost that is released together with 2.0. It makes less sense if this is part of our docs. Consider a new user that picked up Daml only from >= 2.0: this section wouldn’t really make sense to them.

Reply:
The title of the section here makes it pretty clear imo that people who did not use Daml prior to 2.0 can skip it, so I think it makes sense.

Reply:
if we want to keep it maybe consider moving it to the end in something like a "migrating from 1.18" section

Reply:
Moving it down does make sense, yes.

Reply:
I went ahead and removed it; there's nothing in that section about migrating specifically that isn't addressed elsewhere.